app/src/main/java/com/example/notesapp/data/local/note/model/NoteModel.kt

```diff
@@ -0,0 +1,23 @@
+package com.example.notesapp.data.local.note.model
+
+// We should use two different models for the flows,
```

Comment:
What are the flows in this context?

Reply:
In this context, flow is a path of business logic, for example save note flow, update note flow, delete note flow or use case